### DISCUSSION TITLE
does this calender support persian date and time? if not is there any plan to add it in future releases?

### DISCUSSION THREAD

**Discussion by FardinFarnezhad** [ME-MarvinE/XCalendar#164] [Q&A] [answered]
title.

**Comment by ME-MarvinE** [comment, marked as answer]
Hi @FardinFarnezhad ,

To change the DayNames to Persian, set CultureInfo.CurrentUICulture to a Persian culture. I tested with "fa-IR".
To change the Month names to Persian, set CultureInfo.CurrentCulture to a Persian culture. I tested with "fa-IR".
However the displayed year does not change to the Persian year. XCalendar only supports the Gregorian Calendar.

Best to do this before `InitializeComponent()` in App.caml.cs. More info on localisation is in the [Displaying Dates](https://github.com/ME-MarvinE/XCalendar/wiki/Displaying-Dates) page of the wiki.
